src/main/fc/core.c

```diff
@@ -404,6 +396,14 @@ void tryArm(void)
             }
             return;
         }
+
+#ifdef USE_DSHOT_TELEMETRY
+        // send the command to enable DSHOT telemetry
+        if (isMotorProtocolDshot() && motorConfig()->dev.useDshotTelemetry) {
+            pwmWriteDshotCommand(255, getMotorCount(), DSHOT_CMD_SIGNAL_LINE_CONTINUOUS_ERPM_TELEMETRY, true);
```

Comment:
Is there any reason to write the `DSHOT_CMD_SIGNAL_LIME_TELEMETRY_DISABLE` cmd when `! motorConfig()->dev.useDshotTelemetry`?

Reply:
In the normal scenario the power on state of the ESC's will be with the telemetry disabled so we should only have to turn it on if we need it. I guess there's an edge case where the user has the rig battery powered and dshot telemetry enabled. Then while leaving the battery connected, connects to USB and turns `dshot_bidir = OFF` and save/reboots. The ESC's will still have telemetry enabled (I think). Not sure if this would matter though. Any thoughts @joelucid ?

Reply:
Also the emphasis for not blindly issuing the "OFF" command is not about efficiency or eliminating commands - it's more about the fact the anybody who has `dshot_bidir = OFF` likely has ESC's that don't have a supporting firmware (basically everybody but 2 people at the moment). So the effect of issuing the command is not fully known. Probably handled okay but was trying to avoid a possible risk.

Reply:
Maybe we could store a `bidiectionalDshotEverEnabled` (or suchlike) variable, and only send the command to disable it if this is true?

Reply:
I don't think there's an issue with sending unimplemented dshot commands to an ESC. At least that's a reasonable assumption given no related issues have come up yet or when we implemented beacon. So on second I actually think its better to avoid that edge case by always sending the command. If that causes issues we can revisit and consider additional state like ``bidiectionalDshotEverEnabled`` which carries the risk of not always being consistent with the esc state (e.g. brown out).